src/Files.App/UserControls/StatusCenter.xaml

```diff
@@ -740,8 +68,314 @@
+
+						<!--  Progress & Footer  -->
+						<Grid
+							x:Name="StatusCenterItemRichProgressDisplayPanel"
+							Grid.Row="1"
+							Margin="0,8,0,8"
+							ColumnSpacing="4"
+							Visibility="{x:Bind IsInProgress, Converter={StaticResource BoolToVisibilityConverter}, Mode=OneWay}">
+							<Grid.RowDefinitions>
+								<RowDefinition Height="Auto" />
+								<RowDefinition Height="Auto" />
+							</Grid.RowDefinitions>
+							<Grid.ColumnDefinitions>
+								<ColumnDefinition Width="*" />
+								<ColumnDefinition Width="Auto" />
+							</Grid.ColumnDefinitions>
+
+							<!--  Extended ProgressBar  -->
+							<ProgressBar
+								x:Name="MainProgressBar"
+								Margin="4,4,4,4"
+								VerticalAlignment="Top"
+								Foreground="{ThemeResource AppThemeFillColorAttentionBrush}"
+								IsIndeterminate="{x:Bind IsIndeterminateProgress, Mode=OneWay}"
+								Visibility="{x:Bind IsExpanded, Converter={StaticResource NegateBoolToVisibilityConverter}, Mode=OneWay}"
+								Value="{x:Bind ProgressPercentage, Mode=OneWay}" />
+
+							<!--  Graph  -->
+							<Border
+								x:Name="MainGraphCartesianChartClipGrid"
+								Height="72"
+								Margin="4,4,4,12"
+								VerticalAlignment="Top"
+								BorderBrush="{ThemeResource ControlStrongStrokeColorDefaultBrush}"
+								BorderThickness="1"
+								CornerRadius="4"
+								Visibility="{x:Bind IsExpanded, Converter={StaticResource BoolToVisibilityConverter}, Mode=OneWay}">
+
+								<Grid>
+									<ProgressBar
+										x:Name="SecondaryProgressBar"
+										Height="70"
+										MinHeight="70"
+										VerticalAlignment="Stretch"
+										CornerRadius="4,0,0,4"
+										Foreground="{ThemeResource AppThemeFillColorAttentionBrush}"
+										IsIndeterminate="False"
+										Style="{StaticResource CustomProgressBarStyle}"
+										Visibility="{x:Bind IsExpanded, Converter={StaticResource BoolToVisibilityConverter}, Mode=OneWay}"
+										Value="{x:Bind ProgressPercentage, Mode=OneWay}" />
+
+									<livecharts2:CartesianChart
+										x:Name="MainCartesianChart"
+										Height="48"
+										Margin="0,-2"
+										VerticalAlignment="Bottom"
+										AnimationsSpeed="00:00:00.001"
+										Series="{x:Bind SpeedGraphSeries, Mode=OneWay}"
+										TooltipPosition="Hidden"
+										XAxes="{x:Bind SpeedGraphXAxes}"
+										YAxes="{x:Bind SpeedGraphYAxes}" />
+
+									<StackPanel Margin="8,4">
+
+										<TextBlock
+											x:Name="OperationSpeedStringTextBlock"
+											HorizontalAlignment="Right"
+											VerticalAlignment="Top"
+											FontSize="10"
+											Foreground="{ThemeResource TextFillColorSecondaryBrush}"
+											Style="{StaticResource CaptionTextBlockStyle}"
+											Text="{helpers:ResourceString Name=SpeedWithColon}" />
+
+										<TextBlock
+											x:Name="OperationSpeedPerSecTextBlock"
+											HorizontalAlignment="Right"
+											VerticalAlignment="Top"
+											FontSize="13"
```

Comment:
please remove? the style should deal with this :)

Reply:
As above.

Reply:
It's just creating a more inconsistent UI, with all those different sizes